src/Dataverse/MSBuildTasks/Props/SolutionFileItems.props

```diff
@@ -26,4 +26,11 @@
         <EnvironmentVariables Include="$(DataverseSolutionSourceFolderFullPath)\environmentvariabledefinitions\**\environmentvariabledefinition.xml"/>
         <ResxFiles Include="$(DataverseSolutionSourceFolderFullPath)\Resources\**\*.resx"/>
     </ItemGroup>
+    <ItemGroup>
+        <SolutionXmlIntermediateOutputPath Include="$(SolutionPackagerMetadataWorkingDirectory)\Other\Solution.xml"/>
+        <PluginAssemblyMetadataFilesIntermediateOutputPath Include="$(SolutionPackagerMetadataWorkingDirectory)\PluginAssemblies\**\*.dll.data.xml"/>
+        <WorkflowMetadataIntermediateOutputPath Include="$(SolutionPackagerMetadataWorkingDirectory)\Workflows\**\*.xaml"/>
+        <SdkMessageProcessingStepsIntermediateOutputPath Include="$(SolutionPackagerMetadataWorkingDirectory)\SdkMessageProcessingSteps\*.xml"/>
+        <CustomControlsIntermediateOutputPath Include="$(SolutionPackagerMetadataWorkingDirectory)\CustomControls"/>
```

Comment:
I think the folder is called Controls
![image](https://github.com/user-attachments/assets/406c7b46-895f-4869-bd6e-702380c268c8)

Reply:
Removed the unused Item. HAJ solved it like this:  

![image](https://github.com/user-attachments/assets/f615eb49-7695-4800-867a-d5111687b858)

![image](https://github.com/user-attachments/assets/de3efc41-e000-4a86-8e4b-3b87f2eb27d6)